OctopusDSC/Tests/OctopusDSCHelpers.Tests.ps1

```diff
@@ -88,24 +90,37 @@ Describe "Invoke-OctopusServerCommand" {
             "--instance", "OctopusServer",
             "--connectionstring", "Data Source=mydbserver;Initial Catalog=Octopus;Integrated Security=SSPI;Max Pool Size=200;")
 
-        It "Doesn't leak the master key by parroting the command" {
+        It "Doesn't try to mask output when no sensitive values exist " {
+            Invoke-OctopusServerCommand $npkargs
+            Assert-MockCalled Get-MaskedOutput -times 0
+        }
+
+        It "Tries to mask the master key" {
             Invoke-OctopusServerCommand $dbargs
-            Assert-MockCalled Write-Verbose -times 0
```

Comment:
I've been checking the build logs as they go past, not leaky thus far. 

`Assert-MockCalled Write-Verbose -times 0` isn't valid any more anyway, because we're calling it either way, but I could possibly do a mock that checks the `-Message` param value to see if there's a string of ***s

Reply:
yeah - I was thinking about checking the `-Message`

Reply:
Done.